Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Base on offical Node.js Alpine image
+FROM node:current-alpine AS base
+
+# Set working directory
+WORKDIR /base
+
+# Copy package.json and package-lock.json before other files
```

Comment:
```suggestion
# Copy package.json and yarn-lock.json to [SOME_PATH] for installation
```

Reply:
I'd like to explain that "we have to copy package.json and package-lock.json before other files and utilise Docker cache to save re-installing dependencies if unchanged"

Reply:
How about, **we only copy package.json and yarn-lock.json for utilizing Docker cache ...**